Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -6553,6 +6570,24 @@ These ItemGrids are available in the API and can be manipulated by the plugins,
 					},
 					Notes = "Adds the specified amount to the item count. Returns self (useful for chaining).",
 				},
+				AddEnchantmentsFromItem =
+				{
+					Params =
+					{
+						{
+							Name = "Sacrifice",
```

Comment:
Went with Additive, since ToAdd to me implies that it's merging the items themselves (whereas it's just using the enchantments as an additive).